odra-casper/backend/src/utils.rs

```diff
@@ -115,7 +116,7 @@ pub fn install_contract(
 
     // Create new contract.
     if is_upgradable {
-        let access_uref_key = format!("{}_access_token", package_hash_key);
+        let access_uref_key = format(format_args!("{}_access_token", package_hash_key));
```

Comment:
Why?

Reply:
Ok, `fomat!` can be imported from `alloc`